### PR TITLE
Add member Megan

### PR DESCRIPTION
**Pull request by amandhawb**
## Overview

Adding one more Latina Dev! 😄 

## Relevant Issue

https://github.com/Latina-Dev/latina-dev/issues/57

> ✅️ By submitting this PR, I have verified the following

- [x] Reviewed the [contributing guidelines](https://github.com/Latina-Dev/latina-dev/blob/master/.github/CONTRIBUTING.md) 🔍️
- [x] Added my name to the bottom of the list under the **Contributors** section in the [README.md](https://github.com/Latina-Dev/latina-dev/blob/master/README.md) with a link to my personal website or GitHub profile 👥️

### CODE REVIEW

**Comment by FrancesCoronel** [comment]
Otherwise looks good!

**Comment by EllaCodes4** [comment]
> Can we remove the change where the analytics component is moved slightly? It's not related and I think prettier would move it back anyways in next diff

Megan's photo looks stretched out on my end. @amandhawb: Can you resize to a 250px square and see if that looks better? The image is rectangular right now and I think that's why it. Thank you!

**Comment by amandhawb** [comment]
> > Can we remove the change where the analytics component is moved slightly? It's not related and I think prettier would move it back anyways in next diff
> 
> Megan's photo looks stretched out on my end. @amandhawb: Can you resize to a 250px square and see if that looks better? The image is rectangular right now and I think that's why it. Thank you!

I resized it @EllaCodes4 , not sure if it changed because of the cache... I am still trying to see how to configure it

File: public/img/members/megan-rodriguez.jpg
Comment (FrancesCoronel) [inline public/img/members/megan-rodriguez.jpg:1]: you need to resize this image to be square (e.g. 250px by 250px)

**Review by FrancesCoronel — CHANGES_REQUESTED** [review]

**Comment by FrancesCoronel** [comment]
> Can we remove the change where the analytics component is moved slightly? It's not related and I think prettier would move it back anyways in next diff

This was done, thank you! Just made two comments on the image + file name.

File: data/members/megan-rodriguez-.md
[the remark this answers is not in the document]
Comment (EllaCodes4) [reply data/members/megan-rodriguez-.md:1]: @amandhawb Yes, the photo cache has been temperamental. I wonder: Have you tried clearing the cache? If that doesn't work, maybe we should just delete Megan's profile and picture and try adding again!

[the remark this answers is not in the document]
Comment (amandhawb) [reply data/members/megan-rodriguez-.md:1]: @EllaCodes4 I follow this https://stackoverflow.com/questions/68444505/how-to-clear-delete-cache-in-nextjs to clear the cache.. it doesn't change 😢 I will keep investigating, but if it took so long I will delete Megan's profile and add again as you suggested.

**Comment by FrancesCoronel** [comment]
image is low res but otherwise looks good!

**Review by FrancesCoronel — APPROVED** [review]

**Pull request merged**